Makefile

```diff
@@ -0,0 +1,30 @@
+# default task since it's first
+.PHONY: all
+all: build lint test
+
+.PHONY: build
+build:
+	go build -o kubectl-resource_capacity main.go
```

Comment:
using this binary format because it can be added to users path and used locally which is great for dog-fooding a PR